build/test.go

```diff
@@ -25,15 +25,15 @@ const (
 	fixturesDir         = "fixtures"
 )
 
-func (d *Driver) Test(bblfshdVers, image string) error {
+func (d *Driver) Test(bblfshdVers, image string, bench bool) error {
```

Comment:
The abrasive D guy (me) says D has this and it's pretty great to avoid the problem @creachadair warns about: https://dlang.org/library/std/typecons/flag.html

Maybe there is some similar library for Go? (can't find any with a quick search). If not @creachadair would be ok too.

Reply:
I don't know of an existing library for it, but it wouldn't be too hard to write one that could serve this purpose (it's a similar idea to https://godoc.org/bitbucket.org/creachadair/goflags/enumflag, but without the flag plumbing).

I know it's fashionable to complain about Go lacking parametric polymorphism, but the thing I keep bumping into is the lack of compile-time symbolic enumerations.

Reply:
Missed the question about "internal". An answer is that nothing except SDK import this directly. Right now only the CLI in SDK imports this function, but there were plans to import it in fixtures tests as well. In any case, the call will remain internal to SDK.